cypress/integration/checkin/checkin.test.spec.js

```diff
@@ -43,7 +45,9 @@ describe('Check In - Actions ', () => {
   });
 
   it('C347631 Check in: Basic check in', { tags: [TestTypes.smoke] }, () => {
```

Comment:
need to work as temUser with permissions(from TC "Permissions: Check In, View loans, View item records, View fees/fines, Create fees/fines")

Reply:
This scope of changes will be done in fixes related to test 'check in.test.spec.js' by Ihor Bohdan